### ISSUE TITLE
Support for Retry-After header GCM

### ISSUE THREAD

**Issue by SimonV** [NicosKaralis/pushmeup#22]
GCM documentation says that 5xx range responses can contain Retry-After response header with time to wait before next request.

Is it possible to add some sort of indication that this header was returned and pass the Retry-After value to the caller of the gem?

The relevant documentation : http://developer.android.com/google/gcm/http.html

**Comment by NicosKaralis** [comment]
Possible to implement it is, but we would need extra libs and gems

Do we really want some "auto retry-er"? this would involve some delayed jobs that i don't see relevant and would possibly make dependencies that is pretty much project specific, like delayed_job or resque? if i have already resque why the gem is using delayed_job.

I could think in some extension gem, like 'pushmeup_resque' or something but in this gem i would rather leave without this functionality.

tl;dr : is it really important for you? do you think a separate gem to handle that would be too bad? you tell me and i see what i will do

**Comment by SimonV** [comment]
This is no longer important for me personaly. 

What I had in mind was not an implementation of an auto retry mechanism, that would indeen be over complicated. I just wanted the Auto-Retry value returned by GCM on the header to be passed to the caller of the gem. Right now it's impossible to access from outside the gem and it's being ignored.
